### PR TITLE
Allow to pass list of URLs via env and command line

### PR DESCRIPTION
**Pull request by haiodo**
Allow to pass list of URLs via env and command line

Signed-off-by: Andrey Sobolev <haiodo@gmail.com>

### CODE REVIEW

File: pkg/tools/flags/url_slice_value.go [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+func NewURLSliceValue(p *[]*url.URL) pflag.Value {
+	v := &urlSliceValue{
+		values:  p,
+		changed: false,
+	}
+	*p = []*url.URL{}
+	return v
+}
+
+func readAsCSV(val string) ([]*url.URL, error) {
```
Comment (edwarnicke) [inline pkg/tools/flags/url_slice_value.go:50]: Interesting... by reading as csv you basically make both the apparent unitary case:
unix://foo
And the multiple case
unix://foo,10.10.10.1:9111 valid.  Correct?

Comment (haiodo) [reply pkg/tools/flags/url_slice_value.go:50]: Yes.

**Comment by haiodo** [comment]
@edwarnicke PR depend on fix in https://github.com/networkservicemesh/sdk/pull/234 to prevent concurrency from happening. Could try to return tests to pass.

**Comment by haiodo** [comment]
@fkautz could you please check, thanks.

**Pull request merged**